x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/rank/textsimilarity/TextSimilarityRankRetrieverBuilder.java

```diff
@@ -52,6 +55,8 @@ public class TextSimilarityRankRetrieverBuilder extends CompoundRetrieverBuilder
     public static final ParseField FAILURES_ALLOWED_FIELD = new ParseField("allow_rerank_failures");
     public static final ParseField SNIPPETS_FIELD = new ParseField("snippets");
     public static final ParseField NUM_SNIPPETS_FIELD = new ParseField("num_snippets");
```

Comment:
Let's adopt the chunk naming. Snippets is confusing if we expose a chunking strategy.